File: CoreJava/employee-polymorphic-app/src/com/techlab/employee/Developer.java
package com.techlab.employee;

public class Developer extends Employee {
	private final float BONUSPERCENT = 30; 
	public Developer(int empId, String empName) {
		super(empId, empName);
	}

	public float getBONUSPERCENT() {
		return BONUSPERCENT;
	}
	
	
	@Override
	public void calculateSalary() {
		grossSalary = BASICPAY+ (BASICPAY*(BONUSPERCENT/100));
	}
	
	
}

File: C#Basics/Account-Up-DownCasting-App/Account-Up-DownCasting-App/Developer.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Account_Up_DownCasting_App
{
    class Developer : Employee
    {
		private const float BONUSPERCENT = 30;

		public Developer(int empId, String empName):base(empId, empName) { }


		public float getBONUSPERCENT()
		{
			return BONUSPERCENT;
		}


		
		public override void calculateSalary()
		{
			grossSalary = BASICPAY + (BASICPAY * (BONUSPERCENT / 100));
		}
	}
}
